Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -1279,7 +1279,11 @@ class HTTPClientTests: XCTestCase {
                         }
                     } catch {
                         // ok, we failed, pool probably shutdown
-                        XCTAssertEqual(.cancelled, error as? HTTPClientError)
+                        if let clientError = error as? HTTPClientError, clientError == .cancelled || clientError == .alreadyShutdown {
```

Comment:
@weissi Do you agree this makes sense to also sometimes get `alreadyShutdown` here?

Reply:
I’m totally cool with that